<launch>

  <node pkg="joy" type="joy_node" name="JoyStick" >
      <param name="dev" type="string" value="/dev/input/js0" />
  </node>

  <node pkg="joy_controller" type="joy_controller_node" name="joyControler" >
  </node>

</launch>
	
